src/services/legal-services.ts

```diff
@@ -202,6 +202,14 @@ export default class LegalServices {
         if (directors) {
           // convert director list to org-person list
           return directors.map(director => {
+            // WORK-AROUND WARNING !!!
+            // convert directors from "middleInitial" to "middleName"
+            const middleInitial = director.officer['middleInitial']
+            if (middleInitial !== undefined) {
+              director.officer.middleName = middleInitial
+              delete director.officer['middleInitial']
+            }
```

Comment:
It turns out this workaround is needed for Alteration / LimitedRestorationExtension / LimitedRestorationToFull / SpecialResolution filings.

Reply:
Sample alteration:

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/042be880-8a42-4499-bb6b-9be86154b1a9)